src/renderer/hooks/useProcessQr.ts

```diff
@@ -93,103 +180,76 @@ export default function useProcessQR() {
         return
       }
 
-      if (checkQr.kind === 'account' || checkQr.kind === 'login') {
-        if (!skipLoginConfirmation) {
-          // ask if user wants it
-          const is_singular_term =
-            (await BackendRemote.rpc.getAllAccountIds()).length == 1
-
-          const message: string =
-            checkQr.kind === 'account'
-              ? is_singular_term
-                ? 'qraccount_ask_create_and_login'
-                : 'qraccount_ask_create_and_login_another'
-              : checkQr.kind === 'login'
-                ? is_singular_term
-                  ? 'qrlogin_ask_login'
-                  : 'qrlogin_ask_login_another'
-                : '?'
-
-          const replacementValue =
-            checkQr.kind === 'account'
-              ? checkQr.domain
-              : checkQr.kind === 'login'
-                ? checkQr.address
-                : ''
-
-          const userConfirmed = await openConfirmationDialog({
-            message: tx(message, replacementValue),
-            confirmLabel: tx('login_title'),
-          })
-
-          if (!userConfirmed) {
-            return
-          }
-        }
+      // DCACCOUNT: Ask the user if they want to create a new profile on the
+      // given chatmail instance
+      if (qr.kind === 'account') {
+        await startInstantOnboarding(accountId, parsed)
+        callback && callback()
+        return
+      }
 
-        if (screen !== Screens.Welcome) {
-          // log out first (not needed anymore as selectAccount does this automatically now)
-          window.__selectAccount(await BackendRemote.rpc.addAccount())
+      // DCLOGIN: Ask the user if they want to login with given credentials
+      if (qr.kind === 'login') {
+        await startInstantOnboarding(accountId, parsed)
```

Comment:
does not work, also I don't think this needs the instant Onboarding screen, just the dialog asking whether you want to login with those credentials (if account is un-configured ask wether you want to login, if it is configured ask if it should create another account and login there; just as it was previously.)

Reply:
I brought back the original code now, that should hopefully reflect the correct flow now for `DCLOGIN`

Reply:
we could also have the onboarding screen, sounds like @r10s did that for iOS and android?
Anyways however the UI flow, the main thing that I care about is that it works nicely and at all in both cases (scanning from unconfigured account and scanning from configured account).